main.nf

```diff
@@ -298,6 +298,14 @@ deseq2_script_improve_deseq_table = Channel.fromPath( workflow.projectDir + '/bi
 deseq2_id_type_ch = Channel.value(params.feature_id_type)
 species2prefix = Channel.fromPath( workflow.projectDir + '/assets/ens_species_mapping.tsv', checkIfExists: true)
 
+/*
+* Downstream analysis
+*/
+piano_script = Channel.fromPath( workflow.projectDir + '/bin/piano.R', checkIfExists: true )
```

Comment:
hm, or does this not work bc/ you have to call the script like that 

```bash
R CMD BATCH --no-save --no-restore '--args c(".") c("${resFold05}") c("${species}") c("${id_type}") c("${task.cpus}")' ${piano_script}
```

could be... I remember this was sometimes difficult

Reply:
I'd need to check I didn't know about the automatic access to bin.. Just did it like that because this was how the deseq2.R script was also invoked before.

Reply:
I think it's fine like that actually. Normally what you can do in nf is smt like here: 

https://github.com/EBI-Metagenomics/emg-viral-pipeline/blob/master/nextflow/modules/rename.nf#L18

As you can see, the `rename.py` script that is in the `bin` folder can just be accessed by nf. However, I think, when you call a script w/ something like `python3 rename.py` or like in this case here also not at the beginning of a command that does not work.

Reply:
Ah good to know!